src/main/kotlin/com/dota/tracker/dashboard/controller/DashboardApiController.kt

```
@@ -9,11 +12,63 @@ import org.springframework.web.bind.annotation.RestController
+
+        cacheManager.cacheNames.forEach { cacheName ->
+            val cache = cacheManager.getCache(cacheName)
+            when (cache) {
+                is CaffeineCache -> {
+                    val caffeineStats = cache.nativeCache.stats()
+                    val estimatedSize = cache.nativeCache.estimatedSize()
+
+                    stats[cacheName] = mapOf(
+                        "hitCount" to caffeineStats.hitCount(),
+                        "missCount" to caffeineStats.missCount(),
+                        "hitRate" to String.format("%.2f%%", caffeineStats.hitRate() * 100),
```

detekt (1.23.8)

[warning] String.format("%.2f%%", caffeineStats.hitRate() * 100) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)